omnibus/Gemfile

```diff
@@ -8,7 +8,7 @@ gem 'bundler', '>1.10'
 # Install omnibus software
 group :omnibus do
   gem 'omnibus', git: 'https://github.com/chef/omnibus'
-  gem 'omnibus-software', git: 'https://github.com/chef/omnibus-software'
+  gem 'omnibus-software', git: 'https://github.com/chef/omnibus-software', :branch => 'lbaker/land-erlang20'
```

Comment:
We should get this branch merged into omnibus-software so that it can point to master again. Could you create a PR with these changes and an issue to track changing this back to master?

Reply:
https://github.com/chef/omnibus-software/pull/1107

Reply:
https://github.com/chef/omnibus-software/issues/1121